### PR TITLE
Update cookie-related OWNERS and READMEs

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
@mikewest, what would be some other good owners for this?

Can you add a README.md explaining which specs this covers? Is it just https://w3c.github.io/webappsec-csp/cookies/, or also https://tools.ietf.org/html/rfc6265?

Finally, I noticed that https://github.com/w3c/web-platform-tests/pull/4423 deleted some of your tests. Did you know, and is that OK? If it's not OK, then @Hexcles or I should do some kind of audit for tests that we've exported and then had removed.

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-10-01 18:48:21
Finished: 2017-10-01 18:52:26


<h2>Failing Jobs</h2>
<ul>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19000)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/281987425)

**Comment by Hexcles** [comment]
#4423 is a Gecko export of https://bugzilla.mozilla.org/show_bug.cgi?id=976073. 

Looks like the tests were deleted unintentionally and later restored in https://bugzilla.mozilla.org/show_bug.cgi?id=1316532, which was not upstreamed. Could you take a look @jgraham ?

**Comment by foolip** [comment]
@Hexcles, @jgraham is away during October, so I guess we'll need to fix this. Maybe noticing this kind of thing could be part of the audit script, where if files we've exported are deleted by something other than a new export are flagged for review. Most will be fine, but some might be accidents.

For now I'll revert the files back into existence.

**Comment by foolip** [comment]
Now I'm also wondering about the relationship between this and cookie-store/. @bsittler, do you know?

**Comment by bsittler** [comment]
These appear to be tests (originally by @tefn3849 I believe - are you a suitable owner?) verifying behavior specified in https://datatracker.ietf.org/doc/draft-ietf-httpbis-cookie-alone/ by @mikewest 

It bears only tangential relationship to the https://github.com/WICG/cookie-store proposal is that the latter (a proposed new JS API for cookie operations) mentions enforcing the restrictions of the former too, in that case with promise rejections when non-conforming operations are attempted.

**Comment by bsittler** [comment]
This is a wild guess, but perhaps @annevk could knows of a suitable owner for these tests?

**Comment by annevk** [comment]
I would have suggested @bsittler and @mikewest...

At some point @inikulin was interested (see https://github.com/whatwg/html/issues/804), but not sure if that's still the case.

**Comment by inikulin** [comment]
I always willing to help. What are duties with this one?

**Comment by bsittler** [comment]
FWIW I am also willing to help, but like @inikulin I would like to understand what's involved. I actually misunderstood earlier and thought you were asking about who wrote the tests

**Comment by pwnall** [comment]
@foolip I can help, sign me up.

I skimmed through the tests. I'm reasonably familiar with the matter, having read the relevant RFCs in order to implement Chrome's Async Cookies API. I also had to / will have to write or review similar tests for Async Cookies.

**Comment by annevk** [comment]
@inikulin @bsittler ideally OWNERS review incoming tests for the directory they're owner for and address outstanding issues with those tests. As part of this you'll get automatically pinged for each new PR against that directory.

**Comment by foolip** [comment]
OK, I've tried to write sensible OWNERS and README.md for both cookie-related directories now. I added just the people I could see in git log. @inikulin, should I add you somewhere as well?

Can @bsittler, @pwnall or @mikewest review this now? As long as it's improvement that's good enough.

File: cookies/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This directory contains tests for
+[Leave Secure Cookies Alone](https://tools.ietf.org/html/draft-ietf-httpbis-cookie-alone-01).
```
Comment (pwnall) [inline cookies/README.md]: Seems like https://github.com/w3c/web-platform-tests/blob/master/cookies/path/match.html covers a bit more than the IETF draft you linked. I think this is a reasonable first step and we can rely on @mikewest's guidance for the next step 😸

**Review by pwnall — APPROVED** [review]
Thank you very much, especially for the update to cookie-store/README.md

**Comment by pwnall** [comment]
I'll wait for a response from @inikulin for a bit, and then merge. @foolip - feel free to merge if you want to get this off your cache.

**Comment by mikewest** [comment]
I was imagining that we'd eventually migrate the IETF test suite for the cookies RFC into this repository as part of putting together RFC6265bis. I am astoundingly slow at that (and it's currently impossible to test `SameSite` because of https://github.com/w3c/web-platform-tests/issues/2669), but it seems like a reasonable long-term goal.

**Review by mikewest — APPROVED** [review]

**Comment by inikulin** [comment]
@pwnall @foolip You can count me in. I'd like to work on IETF suite port at some point.

**Comment by foolip** [comment]
@mikewest, since you approved I take it you think the README is OK even though it'll hopefully be made inaccurate. If there's a list of RFCs that should be covered by this directory, please update :)

@inikulin, I'll add you to cookies/OWNERS.

**Pull request merged**